### DISCUSSION TITLE
Issue with local setup for contributing

### DISCUSSION THREAD

**Discussion by ktssr** [sequelize/sequelize#15948] [Help & Questions] [answered]
I'm facing an issue after running yarn install command. The error message is as follows:

error /Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/ibm_db: Command failed.
Exit code: 1
Command: node installer/driverInstall.js
Arguments: 
Directory: /Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/ibm_db
Output:

I'm not sure what could be causing this issue, so I'm hoping someone can provide some guidance on how to resolve it. Any help would be greatly appreciated. Thank you in advance!

**Comment by WikiRik** [comment]
Was there anything in the output? Most likely it's just a temporary issue with `ibm_db` but I'm not sure. 

The easiest fix would be to not install `ibm_db` through removing it from here; https://github.com/sequelize/sequelize/blob/811a06d17a62f3efa722f17fb856107241c9922d/packages/core/package.json#L93

Only thing this should affect is when you try to run the tests for DB2, but they run on the CI either way

**Reply by ktssr** [reply]
No nothing in output. I removed ibm_db from package.json. Getting different error now 
CXX(target) Release/obj.target/odbc/src/odbc.o
clang: error: no such file or directory: 'source/sequelize/node_modules/node-addon-api'
make: *** [Release/obj.target/odbc/src/odbc.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/node-gyp/lib/build.js:203:23)
gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 22.4.0
gyp ERR! command "/usr/local/bin/node" "/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc/lib/bindings/napi-v6/odbc.node" "--module_name=odbc" "--module_path=/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc/lib/bindings/napi-v6" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=6" "--node_napi_label=napi-v6"
gyp ERR! cwd /Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc
gyp ERR! node -v v18.16.0
gyp ERR! node-gyp -v v9.3.1
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc/lib/bindings/napi-v6/odbc.node --module_name=odbc --module_path=/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc/lib/bindings/napi-v6 --napi_version=8 --node_abi_napi=napi --napi_build_version=6 --node_napi_label=napi-v6' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1091:16)
node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:302:5)
node-pre-gyp ERR! System Darwin 22.4.0
node-pre-gyp ERR! command "/usr/local/bin/node" "/Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/tejasaisandeepreddykonala/Documents/open source/sequelize/node_modules/odbc
node-pre-gyp ERR! node -v v18.16.0
node-pre-gyp ERR! node-pre-gyp -v v1.0.10
node-pre-gyp ERR! not ok

**Reply by WikiRik** [reply, marked as answer]
Same fix, but then for `odbc` in package.json

That controls the `ibmi` dialect which we only run unit tests on and is still experimental
